### ISSUE TITLE
curly bracket in .txt file 

### ISSUE THREAD

**Issue by bea-alex** [nlplab/brat#1300]
Just to make other users aware that brat didn't behave as it should for a .txt file containing a curly bracket in particular when annotating across lines.

Entity annotation seems to work fine for strings containing a curly bracket as long a the entity does not span across lines.  As soon as it does then the visualisation either appears wrong or the annotation window collapses completely.

In my case the curly brackets are OCR errors, so I'll preprocess the text to fix them but in other cases that might not be possible.

**Comment by anderscui** [comment]
Yes, this issue happens also when whitespace characters are at the beginning of line. I have to pre-process the lines and then revert them.
